web/profiles/sdd/sdd.info.yml

```diff
@@ -53,6 +53,7 @@ dependencies:
   - menu_link_attributes
   - menu_link_content
   - menu_ui
+  - migrate_generator
```

Comment:
It's weird that you added it here to profile as it requires additional config exported and could lead to inconsistency on subsequent config exports